remote-messaging/remote-messaging-impl/src/main/java/com/duckduckgo/remote/messaging/impl/RemoteMessagingConfigProcessor.kt

```diff
@@ -41,6 +42,7 @@ class RealRemoteMessagingConfigProcessor(
 
         if (isNewVersion || shouldProcess) {
             val config = remoteMessagingConfigJsonMapper.map(jsonRemoteMessagingConfig)
+            val message = remoteMessagingConfigMatcher.evaluate(config)
```

Comment:
Good, `config` was used here, now `message` is unused...but I am sure it'll be in the upcoming PRs :D